source/AI.cpp

```diff
@@ -3255,20 +3363,10 @@ void AI::MovePlayer(Ship &ship, const PlayerInfo &player, Command &activeCommand
 					found = true;
 				}
 			}
-		// If no ship was found, look for nearby asteroids.
-		double asteroidRange = 100. * sqrt(ship.Attributes().Get("asteroid scan power"));
-		if(!found && asteroidRange)
-		{
-			for(const shared_ptr<Minable> &asteroid : minables)
-			{
-				double range = ship.Position().Distance(asteroid->Position());
-				if(range < asteroidRange)
-				{
-					ship.SetTargetAsteroid(asteroid);
-					asteroidRange = range;
-				}
-			}
-		}
+		// If no ship was found, look for nearby asteroids.  This functionality is preserving behavior before the mining
+		// shortcut was introduced.
```

Comment:
Unnecessary editorializing 
```suggestion
		// If no ship was found, look for nearby asteroids.
```

Reply:
Awesome, thanks for the good suggestion.